locales/en.yml

```
@@ -28,28 +28,39 @@ myprofile:
   posts:
     add: Write a post
   title: My Profile
-chat:
+  chat:
   title: Chat
-  welcome: Start conversation with something nice.
-  with: Chat with {username}
-  dateDiff: "{diff} ago"
+  conversation: 
+      new: Start conversations and they will appear here.
+     welcome: Start conversation with something nice.
+  message:
+      new: Write a message
+      writer: by you
```

I would call it `byYou` or `own` or something closer to meaning. It's not writer ;)

```suggestion
      byYou: by you
```